frontend/views/containers/NavBar.vue

```diff
@@ -91,7 +91,6 @@
         </div>
       </div>
       <time-travel v-show="ephemeral.showTimeTravel" :toggleVisibility="toggleTimeTravel" />
-    </div>
```

Comment:
Nice catch! And yeah in the future you are also welcome to fix the indentation on the file by re-indenting the entire block left (as it should be). :-)

Reply:
Ok! 👍